web-modeler-openapi-cli/docs/CollaboratorsApi.md

```diff
@@ -0,0 +1,225 @@
+import org.camunda.community.webmodeler.client.invoker.auth.*;
+import org.camunda.community.webmodeler.client.invoker.models.*;
+import org.camunda.community.webmodeler.client.api.CollaboratorsApi;
+
+public class Example {
+  public static void main(String[] args) {
+    ApiClient defaultClient = Configuration.getDefaultApiClient();
+    defaultClient.setBasePath("https://modeler.cloud.camunda.io");
+    
+    // Configure HTTP bearer authorization: Bearer
+    HttpBearerAuth Bearer = (HttpBearerAuth) defaultClient.getAuthentication("Bearer");
+    Bearer.setBearerToken("BEARER TOKEN");
+
+    CollaboratorsApi apiInstance = new CollaboratorsApi(defaultClient);
+    CreateCollaboratorDto createCollaboratorDto = new CreateCollaboratorDto(); // CreateCollaboratorDto | 
+    try {
+      apiInstance.modifyCollaborator(createCollaboratorDto);
+    } catch (ApiException e) {
+      System.err.println("Exception when calling CollaboratorsApi#modifyCollaborator");
+      System.err.println("Status code: " + e.getCode());
+      System.err.println("Reason: " + e.getResponseBody());
+      System.err.println("Response headers: " + e.getResponseHeaders());
+      e.printStackTrace();
+    }
+  }
+}
+```
+
+### Parameters
+
+|           Name            |                         Type                          | Description | Notes |
+|---------------------------|-------------------------------------------------------|-------------|-------|
+| **createCollaboratorDto** | [**CreateCollaboratorDto**](CreateCollaboratorDto.md) |             |       |
+
+### Return type
+
+null (empty response body)
+
+### Authorization
+
+[Bearer](../README.md#Bearer)
+
+### HTTP request headers
+
+- **Content-Type**: application/json
+- **Accept**: application/json
+
+### HTTP response details
+
+| Status code |      Description      | Response headers |
+|-------------|-----------------------|------------------|
+| **204**     | No Content            | -                |
+| **400**     | Bad Request           | -                |
+| **404**     | Not Found             | -                |
+| **500**     | Internal Server Error | -                |
+
+<a name="searchCollaborators"></a>
+
+# **searchCollaborators**
+
+> PubSearchResultDtoProjectCollaboratorDto searchCollaborators(pubSearchDtoProjectCollaboratorDto)
+
+Searches for collaborators.&lt;br/&gt; &lt;ul&gt;   &lt;li&gt;&lt;p&gt;&lt;em&gt;filter&lt;/em&gt; specifies which fields should match. Only items that match the given fields will be    returned.&lt;/p&gt;&lt;/li&gt;   &lt;li&gt;&lt;p&gt;&lt;em&gt;sort&lt;/em&gt; specifies by which fields and direction (&lt;code&gt;ASC&lt;/code&gt;/&lt;code&gt;DESC&lt;/code&gt;) the result    should be sorted.&lt;/p&gt;&lt;/li&gt;   &lt;li&gt;&lt;p&gt;&lt;em&gt;page&lt;/em&gt; specifies the page number to return.&lt;/p&gt;&lt;/li&gt;   &lt;li&gt;&lt;p&gt;&lt;em&gt;size&lt;/em&gt; specifies the number of items per page. The default value is 10.&lt;/p&gt;&lt;/li&gt; &lt;/ul&gt;
```

Comment:
Same with this as above

Reply:
See https://github.com/camunda-community-hub/web-modeler-java-client/pull/19#discussion_r1340501323